### PR TITLE
LPD 39711

### PR DESCRIPTION
**Pull request by ivicac**
https://liferay.atlassian.net/browse/LPD-39711

### CODE REVIEW

**Comment by ivicac** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-faro/osb-faro-service/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -12,17 +12,18 @@ go-to-workspace=Go to Workspace
 invite-teammates-to-collaborate=Invite teammates to collaborate
 log-in-to-your-workspace-to-access-this-property=Log in to your workspace to access this property.
 member-fragment=member
-new-request-to-join-workspace=New Request to Join Workspace
```
Comment (marcosapmf) [inline modules/dxp/apps/osb/osb-faro/osb-faro-service/src/main/resources/content/Language.properties:15]: Hey @ivicac, `Language.properties` changes should be in a different commit than the changes in generated files

**Review by marcosapmf — CHANGES_REQUESTED** [review]

**Review by marcosapmf — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-faro/osb-faro-service/src/main/resources/content/Language.properties [hunk 2]
```diff
@@ -12,17 +12,18 @@ go-to-workspace=Go to Workspace
...
 x-has-been-added-to-your-properties-list={0} has been added to your Properties list.
 x-has-requested-to-join-the-x-workspace={0} has requested to join the {1} workspace.
 x-is-now-ready-you-can-access-it-with-the-link-below={0} is now ready. You can access it with the link below.
-you-have-been-added=You have been added!
 you-have-been-added-as-a-team-x-on-workspace-x=You have been added as a team {0} on workspace {1}.
 your-analytics-cloud-workspace-is-ready=Your Analytics Cloud Workspace is ready
 your-new-workspace-is-ready-to-use=Your new workspace is ready to use
-your-request-is-complete=Your Request is Complete!
\ No newline at end of file
+your-request-is-complete=Your Request is Complete
```
[the remark this answers is not in the document]
Comment (ivicac) [reply modules/dxp/apps/osb/osb-faro/osb-faro-service/src/main/resources/content/Language.properties]: @marcosapmf updated PR

**Comment by ivicac** [comment]
ci:test:sf

**Review by marcosapmf — APPROVED** [review]

**Review by marcosapmf — APPROVED** [review]

**Comment by marcosapmf** [comment]
ci:forward
